src/main/drivers/bus_spi.c

```diff
@@ -240,6 +240,25 @@ void spiBusSetInstance(busDevice_t *bus, SPI_TypeDef *instance)
     bus->busdev_u.spi.instance = instance;
 }
 
+uint16_t spiCalculateDivider(uint32_t freq)
+{
+#if defined(STM32F4) || defined(STM32G4) || defined(STM32F7)
+    uint32_t spiClk = SystemCoreClock/2;
```

Comment:
Betaflight coding style is to have spaces around operators (except inside braces).

Reply:
Whoops. Fixed.